docs/source/package_reference/hf_api.mdx

```diff
@@ -0,0 +1,47 @@
+# Hugging Face Hub API
+
+[[autodoc]] login
+
+[[autodoc]] HfApi.logout
+
+[[autodoc]] HfApi.whoami
+
+[[autodoc]] HfApi.list_models
+
+[[autodoc]] HfApi.model_info
+
+[[autodoc]] HfApi.list_repo_files
+
+[[autodoc]] HfApi.list_datasets
+
+[[autodoc]] HfApi.dataset_info
+
+[[autodoc]] HfApi.list_metrics
+
+[[autodoc]] HfApi.get_model_tags
+
+[[autodoc]] HfApi.get_dataset_tags
```

Comment:
does every method of a class need to be listed her? I would rather if we could just list the class and all public methods are then rendered.

Reply:
Good catch, I actually just wanted to list the methods accessible at the root, not the `HfApi` itself. Will adapt!